compose/runtime/runtime/build.gradle

```diff
@@ -175,6 +175,11 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
                 }
             }
 
+            def jvmAndNativeMain = sourceSets.create("jvmAndNativeMain")
+            jvmAndNativeMain.dependsOn(commonMain)
```

Comment:
maybe `commonNonWebTest` as we named it in another PR?

Reply:
do you mean `commonNonWebMain`, since it's not a test source set?